src/test/kotlin/org/rust/ide/refactoring/RenameTest.kt

```diff
@@ -519,6 +519,46 @@ class RenameTest : RsTestBase() {
         foo! { type T = Spam; }
     """)
 
+    fun `test use initialization shorthand after rename`() = doTest("value", """
+        struct Foo {
+            value: u32
+        }
+        fn bar(/*caret*/foo: u32) -> Foo {
+            Foo {
+                value: foo
```

Comment:
```suggestion
                value: /*comment*/ foo
```
To check that we keep comment here